styles/themes/charcoal.less

```diff
@@ -21,6 +21,8 @@
 @active-color: #ff66aa;
 @sans-serif-font: Verdana, Helvetica, sans-serif;
 
+@graph-series-colors: #1f77b4, #ff7f0e, #2ca02c, #d62728, #9467bd, #8c564b, #e377c2, #7f7f7f, #bcbd22, #17becf;
```

Comment:
It's entirely appropriate for `@graph-series-colors` to be her for `charcoal.less`, but in general, for the standard light themes, we've kept the defaults in `layout.less` (and `page_interfaces.less`), and put overrides in any non-standard themes. Would it be appropriate at some point to move this piece of code into `layout.less` at some point for the other three themes?

Reply:
that makes sense! I was wondering what the rationale was.